converter/src/main/java/gov/cms/qpp/conversion/model/TemplateId.java

```diff
@@ -1,31 +1,32 @@
 package gov.cms.qpp.conversion.model;
 
+import com.google.common.base.Strings;
+
 import gov.cms.qpp.conversion.Context;
+import gov.cms.qpp.conversion.util.EnvironmentHelper;
+
 import java.util.HashMap;
 import java.util.Map;
 import java.util.function.Function;
 
-import com.google.common.base.Strings;
-import gov.cms.qpp.conversion.util.EnvironmentHelper;
-
 /**
  * An enumeration of known templates IDs.
  */
 public enum TemplateId {
```

Comment:
What was your process for determining whether the template id has a human readable title?

Reply:
The `TemplateId` that has a human readable title will have its title filled into the validation error location.

My thought was when it comes to a human wanting to know the location of an error, a measure will be the deepest that they care about.  If they want more information, they can use the line, column, and XPath information.